### ISSUE TITLE
Only small letters in resource name can be found

### ISSUE THREAD

**Issue by spaetz** [nextcloud/calendar_resource_management#44]
Not sure if this is a Calendar issue, or a resource management issue, it could also be related to issue #36.

I added a resource called "Virtual Office Sebastian". When I search for resources in the calendar app, it will show the resource when I enter an "i" but not when I enter a "v" or a "V". In fact, it will find the resource on any small letter that is part of the resource name. It will also not find the resource when I enter "o" or "O" as search term, so it seems to fail the search on capital letters.

I apologize if I should have logged this against the calendar component and would be grateful for a pointer if that were the right place.

**Comment by ChristophWurst** [comment]
Interesting! The search is handled by the CalDAV back-end that lives in https://github.com/nextcloud/server/tree/master/apps/dav

**Comment by q-wertz** [comment]
@spaetz 
I had some progress in my issue. Maybe you can check whether you have a similar behavior https://github.com/nextcloud/calendar/issues/3907#issuecomment-1047560002

Single letters (also capital ones) seem to work fine :sweat_smile:

**Comment by spaetz** [comment]
It is indeed more difficult than it seemed. these are my resources:
php ./occ calendar-resource:resources:list
```
Buildings:
+----+---------+---------+-------------------+-----------------------+
| ID | Name    | Address | Description       | Wheelchair Accessible |
+----+---------+---------+-------------------+-----------------------+
| 1  | Digital | Online  | Digital resources | no                    |
+----+---------+---------+-------------------+-----------------------+
Rooms:
+----+----------------+----------------+------------------------------+---------------------------------+-----------+----------------+----------+-------------+-------+--------------------+----+-----------+------------+-----------------------+
| ID | UID            | Name           | Located in                   | Email                           | Room Type | Contact Person | Capacity | Room Number | Phone | Video Conferencing | TV | Projector | Whiteboard | Wheelchair Accessible |
+----+----------------+----------------+------------------------------+---------------------------------+-----------+----------------+----------+-------------+-------+--------------------+----+-----------+------------+-----------------------+
| 7  | virtual_office | Virtual Office | Digital, Video collaboration | Sebastian.Spaeth@uni-hamburg.de | Videochat | spaetz         | 0        |             | no    | yes                | no | no        | no         | no                    |
+----+----------------+----------------+------------------------------+---------------------------------+-----------+----------------+----------+-------------+-------+--------------------+----+-----------+------------+-----------------------+
Resources:
+----+---------------+---------------------------------+------------+----------------+-----------------+
| ID | UID           | Name                            | Located in | Contact Person | Type            |
+----+---------------+---------------------------------+------------+----------------+-----------------+
| 3  | office_spaeth | Virtual Office Sebastian Spaeth | 1 Digital  |                | Big Blue Button |
+----+---------------+---------------------------------+------------+----------------+-----------------+

```

and it does not find anything searching for "l", but it does find something searching for "S". So it seems that it indeed only matches **rooms** if the letter is in both the resource name and the email address. I am not sure which logic is applied to find "resources" (which don't have an email address attached). In any case, it seems to be a duplicate of https://github.com/nextcloud/calendar/issues/3907 so I will see if the next update fixes that and close this one as duplicate.

**Comment by ChristophWurst** [comment]
We will ship the fix with Calendar v3.1.
